packages/core/src/dispatcher.ts

```diff
@@ -10,6 +10,9 @@ import { DoctypeUtils, RootLogger, Logger, IpfsApi } from "@ceramicnetwork/commo
 import { TextDecoder } from 'util'
 import DocID from "@ceramicnetwork/docid";
 
+const DEFAULT_GET_TIMEOUT = 30000 // 30 seconds
+const DEFAULT_MAX_IPFS_OBJECT_SIZE = 500000 // 500 KB
```

Comment:
The word "default" in my mind implies that it can be overridden.  Given that there's no way to override these currently, they just are what is used, I'd vote to drop the word "default" from the names

Reply:
Can we make the max size an even number? @PaulLeCam suggested 256 KB earlier.

Reply:
Yep 👍